docker/build-tools/Dockerfile

```diff
@@ -88,38 +88,48 @@ ADD https://github.com/protocolbuffers/protobuf/releases/download/v${PROTOC_VERS
 RUN unzip /tmp/protoc-${PROTOC_VERSION}-linux-x86_64.zip
 RUN mv /tmp/bin/protoc ${OUTDIR}/usr/bin
 
+# Install gobin. We use gobin to build all of our tools such that they remain
+# independent of their dependency versions.
+RUN GO111MODULE=off go build -o /usr/local/go/gobin github.com/myitcv/gobin
```

Comment:
```suggestion
RUN GO111MODULE=off go get github.com/myitcv/gobin
```

Reply:
ya I already had this in my local build. Although now mason is not building, and it was building locally.

Reply:
It isn't building for me locally.